### DISCUSSION TITLE
How to enable stack backtrace and debug logging in standalone mode

### DISCUSSION THREAD

**Discussion by J0HN50N133** [GreptimeTeam/greptimedb#4352] [Q&A] [answered]
I've modified `loggin.level` to `debug` in `standalone.example.toml` and started a standalone instance with `cargo run -- standalone start`.
![image](https://github.com/GreptimeTeam/greptimedb/assets/53596783/d4168d63-7e8c-428e-b753-73a66101d691)

However, the actual logging options is as below:
![image](https://github.com/GreptimeTeam/greptimedb/assets/53596783/0c84c513-f46b-475c-8b3c-d1f0c4aec82e)

Then I triggered an error:
![image](https://github.com/GreptimeTeam/greptimedb/assets/53596783/78953217-9a3e-4ece-8997-958ea5f83b9b)

However, the error stacktrace is not printed to console:
![image](https://github.com/GreptimeTeam/greptimedb/assets/53596783/2cb46646-6a3a-4c76-a5a6-1aadeeab2fec)

**Comment by killme2008** [comment]
Need the ` -c config/standalone.example.toml`?

```
RUST_BACKTRACE=full cargo run -- standalone start -c config/standalone.example.toml
```

**Comment by killme2008** [comment, marked as answer]
A better way to set log level by command line arg:

```
RUST_BACKTRACE=full cargo run -- standalone start --log-level=debug -c config/standalone.example.toml
```
